### PR TITLE
[Bullseye] Upgrade sonic-sdk image to bullseye

### PR DESCRIPTION
**Pull request by vivekrnv**
Signed-off-by: Vivek Reddy <vkarri@nvidia.com>

#### Why I did it

Upgrade the app-extension developer environments (sonic-sdk & sonic-sdk-bullseye) to bullseye

#### How to verify it

Built an app-extension using these images and verified if it is up and running.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: rules/docker-sonic-sdk-buildenv.mk [hunk 1]
```diff
@@ -28,5 +28,5 @@ $(DOCKER_SONIC_SDK_BUILDENV)_LOAD_DOCKERS += $(DOCKER_SONIC_SDK)
 SONIC_DOCKER_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV)
 SONIC_DOCKER_DBG_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV_DBG)
 
-SONIC_BUSTER_DOCKERS += $(DOCKER_SONIC_SDK_BUILDENV)
-SONIC_BUSTER_DBG_DOCKERS += $(DOCKER_SONIC_SDK_BUILDENV_DBG)
+SONIC_DOCKER_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV)
```
Comment (saiarcot895) [inline rules/docker-sonic-sdk-buildenv.mk:31]: Should these be `SONIC_BULLSEYE_DOCKERS` and `SONIC_BULLSEYE_DBG_DOCKERS` instead?

Comment (vivekrnv) [reply rules/docker-sonic-sdk-buildenv.mk:31]: I think no, This has to be under SONIC_DOCKER_IMAGES https://github.com/sonic-net/sonic-buildimage/blob/master/slave.mk#L961 && https://github.com/sonic-net/sonic-buildimage/blob/master/slave.mk#L949 

Although, i see some targets were added to SONIC_BULLSEYE_DOCKERS but the variable is not used anywhere. Do you want me to also include this variable for these targets?

Comment (saiarcot895) [reply rules/docker-sonic-sdk-buildenv.mk:31]: `SONIC_DOCKER_IMAGES` is already specified two lines above here (lines 28 and 29).

You're correct in that `SONIC_BULLSEYE_DOCKERS` is currently not used anywhere, and that some of the container upgrade PRs added those containers to this variable. I'm fine either way if it's added here or not.

Comment (vivekrnv) [reply rules/docker-sonic-sdk-buildenv.mk]: Oh, sry. I've overlooked it. Updated

File: rules/docker-sonic-sdk-buildenv.mk [hunk 2]
```diff
@@ -28,5 +28,5 @@ $(DOCKER_SONIC_SDK_BUILDENV)_LOAD_DOCKERS += $(DOCKER_SONIC_SDK)
 SONIC_DOCKER_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV)
 SONIC_DOCKER_DBG_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV_DBG)
 
-SONIC_BUSTER_DOCKERS += $(DOCKER_SONIC_SDK_BUILDENV)
-SONIC_BUSTER_DBG_DOCKERS += $(DOCKER_SONIC_SDK_BUILDENV_DBG)
+SONIC_BULLSEYE_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV)
```
Comment (saiarcot895) [inline rules/docker-sonic-sdk-buildenv.mk:31]: `s/_IMAGES/_DOCKERS/g`

Comment (vivekrnv) [reply rules/docker-sonic-sdk-buildenv.mk]: updated

**Comment by liat-grozovik** [comment]
@saiarcot895 could you please help to approve?

**Review by saiarcot895 — APPROVED** [review]

**Review by dgsudharsan — APPROVED** [review]

**Comment by dgsudharsan** [comment]
@StormLiangMS Please set the label included in 202211 once they are cherry-picked

**Pull request merged**